1_fetch/src/get_sb_data.R

```diff
@@ -272,5 +205,28 @@ calc_avg_wildfire <- function(sb_data) {
     arrange(COMID, label) %>%
     select(COMID, label, avg_annual) %>%
     pivot_wider(names_from = "label", values_from = "avg_annual")
-  return(wildfire)
-}
\ No newline at end of file
+  
+  data <- data %>%
+    select(-ends_with(".y"), -ID, -starts_with("..."), -contains("_S1"), -contains("PHYSIO_AREA"), 
+           -contains("SOHL"), -contains("NDAMS"), -contains("STORAGE"), 
+           -contains("MAJOR"), -contains("TAV"), -contains("PPT"), -contains("WILDFIRE")) %>%
+    left_join(land_cover) %>%
+    left_join(dams) %>%
+    left_join(weather) %>%
+    left_join(wildfire)
+  
+  rename_dup_headers <- list()
```

Comment:
What causes these duplicates? Could we instead `select(-colname)` in the `left_join`s?

Reply:
There is one set of feature variables in Science Base that is duplicated (https://www.sciencebase.gov/catalog/item/57976a0ce4b021cadec97890?community=National+Water-Quality+Assessment+-+Hydrologic+Systems+Team+%28NAWQA+-+HST%29) - you will notice there is a FTYPE_PCT_CONUS.txt in addition to FTYPE_PCT_CONUS.zip (which also contains the same .txt file). Rather than hard-code in Andrew's downloading functions to skip over this duplicate, I think it is easier for us to clean up after. Further, if any other feature variables groups encounter this same problem, the code above should work to remove all the doubles (appended with ".y") and rename the originals (remove the appendage ".x").

Reply:
Okay, thanks for that explanation. I don't see where `remove all the doubles (appended with ".y")` happens. It looks like only the `.x` are renamed and `.y` would be kept

Reply:
`select(-ends_with(".y")`  before the left joins removes the .y, leaving just the .x to be renamed.